src/main/java/rs/teslaris/core/repository/document/PublisherRepository.java

```diff
@@ -1,9 +1,25 @@
 package rs.teslaris.core.repository.document;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 import rs.teslaris.core.model.document.Publisher;
 
 @Repository
 public interface PublisherRepository extends JpaRepository<Publisher, Integer> {
+
+    @Query("select count(d) > 0 from Dataset d join d.publisher p where p.id = :publisherId")
+    boolean publishedDataset(Integer publisherId);
```

Comment:
hasPublishedDataset? To make it consistent with JournalRepository and EventRepository